assets/benchmarks/shaders/Benchmark_PsSkyBox.hlsl

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 Google LLC
```

Comment:
You can put both shaders in the same file, as long as the entrypoint names varies between VS and PS stages (see other shaders).
Btw, maybe you could not add new shaders, remove the `assets/basic/shaders/unlit.hlsl` and reuse `assets/basic/shaders/Texture.hlsl` for this skybox?

Reply:
I put them in one single file and took as an example the `Texture.hlsl`.

Reply:
sorry i am still seeing 2 shader files. maybe the new change is not pushed?

Reply:
I re based this PR into the main branch, but then I had to force the push instead of adding a new commit.
The last commit to review is [7824134](https://github.com/google/bigwheels/commit/7824134bb349740fbaf89430027395dbe889f63e).